lib/monitoring/aws-dynamo/DynamoTableMetricFactory.ts

```diff
@@ -98,7 +98,7 @@ export class DynamoTableMetricFactory {
       'MetricName="SuccessfulRequestLatency"',
       {
         TableName: this.table.tableName,
-        Operation: undefined,
+        Operation: undefined as unknown as string,
```

Comment:
We would need to see if deployment passes with this. Could you try to deploy this change from a test package to see if everything works fine?

Reply:
I deployed it to a test account and it seems to work fine. This is the metric that ends up in the dashboard:

```
    "metrics": [
        [ { "label": " ", "expression": "SEARCH('{AWS/DynamoDB,TableName,Operation} TableName=\"SampleTable\" MetricName=\"SuccessfulRequestLatency\"', 'Average', 300)", "region": "us-east-1" } ]
    ],
```

Which matches up with what's generated with the older monocdk-based codebase.